national_rail/test_load.py

```diff
@@ -0,0 +1,158 @@
+""" Unit tests to test load functions. """
+
+from unittest.mock import MagicMock, patch
+import os
+from datetime import datetime
+
+import psycopg2
+from psycopg2.extensions import connection, cursor
+
+from load import (
+    get_connection,
+    get_cursor,
+    upload_incident,
+    check_if_exists,
+    get_operator_code_id,
+    upload_affected_operator,
+)
+
+
+@patch.dict(
+    os.environ,
+    {
+        "DB_IP": "localhost",
+        "DB_NAME": "test_db",
+        "DB_USERNAME": "test_username",
+        "DB_PASSWORD": "test_password",
+        "DB_PORT": "test_port",
+    },
+)
+@patch("load.connect")
+def test_get_connection(mock_connect: connection):
+    """ Tests get_connection returns connection. """
+
+    mock_connect.return_value = "mock_connection"
+    db_connection = get_connection()
+
+    assert db_connection == "mock_connection"
+    mock_connect.assert_called_once()
+
+
+@patch("load.get_connection")
+def test_get_cursor(mock_get_connection: connection):
+    """ Tests get_cursor returns cursor. """
+
+    mock_connection = MagicMock()
+    mock_get_connection.return_value = mock_connection
```

Comment:
A slight improvement could be making the mock connection and cursors as a class variable and have these tests as one class, so that you don't need to keep repeating these lines. However, it is a big change with small effect so it's up to you if you want to.